applications/StructuralMechanicsApplication/python_scripts/python_solvers_wrapper_structural.py

```diff
@@ -42,9 +42,12 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
         elif solver_type == "adjoint_static":
             solver_module_name = "structural_mechanics_adjoint_static_solver"
 
+        elif solver_type == "custom_scipy":
+            solver_module_name = "structural_mechanics_custom_scipy_solver"
+
         else:
             err_msg =  "The requested solver type \"" + solver_type + "\" is not in the python solvers wrapper\n"
-            err_msg += "Available options are: \"static\", \"dynamic\", \"eigen_value\", \"harmonic_analysis\", \"formfinding\", \"adjoint_static\""
+            err_msg += "Available options are: \"static\", \"custom_scipy\", \"dynamic\", \"eigen_value\", \"harmonic_analysis\", \"formfinding\", \"adjoint_static\""
```

Comment:
I don't think it should be here, 
Reason: this is quite sth special, that cannot be used without a corresponding `AnalysisStage`
Which btw I think you should add in this PR

Reply:
why do you need a special analysis stage? I do think that one should be able to construct it...that would be very good for experiencing

Reply:
my point is how do you use this solver?
It extends the interface of `PythonSolver` with the access-fct to the mass & stiffness matrix
=> this means that you need a custom analysis stage in any case, otherwise it does not make sense using this solver, bcs except building the matrices it does (and should not) do anything

Reply:
as we spoke, this opens the door to experimenting with scipy. one should derive and use his experimental solver